cv_next/components/ui/filters/modeSwitch.tsx

```diff
@@ -0,0 +1,26 @@
+
```

Comment:
The entire `\ui\filters` folder should probably be moved to `\feed\components`. Unless they are used elsewhere as well.